lib/src/mixins/batched_redraws.dart

```diff
@@ -6,23 +6,42 @@ import 'dart:html';
 import 'package:react/react.dart' as react;
 
 class _RedrawScheduler implements Function {
-  Set<react.Component> _components = new Set();
+  Map<react.Component, List<Function>> _components =
+      <react.Component, List<Function>>{};
```

Comment:
Interesting, well then I say keep it and we'll just agree to remember what it means... :-)

As an aside, is the type on the right of the `=` even necessary since it is already present on the left?